sql/migrations/20201120223448_world.sql

```diff
@@ -0,0 +1,431 @@
+
+-- Scroll of Agility - Salazar Bloch
+UPDATE `npc_vendor` SET `maxcount` = 5 WHERE `entry` = 4581 AND `item` = 3012;
+
+-- Pattern: Green Whelp Bracers - Joseph Moore
+INSERT INTO `npc_vendor` (`entry`, `item`, `maxcount`, `incrtime`) VALUES
+(4589, 7451, 1, 7200);
+
+-- Strong Fishing Pole - Lizbeth Cromwell
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 7200 WHERE `entry` = 4574 AND `item` = 6365;
+
+-- Missing Items - Elizabeth Van Talen
+INSERT INTO `npc_vendor` (`entry`, `item`, `maxcount`, `incrtime`) VALUES
+(4587, 4389, 1, 10800),
+(4587, 4404, 3, 10800),
+(4587, 4361, 2, 10800);
+
+-- Teldrassil
+-- Missing Items - Felicia Doan
+INSERT INTO `npc_vendor` (`entry`, `item`) VALUES
+(3608, 3131),
+(3608, 2946);
+
+-- Strong Fishing Pole - Nessa Shadowsong
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 7200 WHERE `entry` = 10118 AND `item` = 6365;
+
+-- Darkshore
+-- Striped Yellowtail - Heldan Galesong
+UPDATE `npc_vendor` SET `maxcount` = 0, `incrtime` = 0 WHERE `entry` = 4307 AND `item` = 21552;
+
+-- Correct Items - Tiyani
+DELETE FROM `npc_vendor` WHERE `entry` = 4195 AND `item` IN (4536, 4537, 4538, 4539, 4602, 8953, 16171);
+UPDATE `npc_vendor` SET `maxcount` = 0, `incrtime` = 0 WHERE `entry` = 4195 AND `item` IN (21030, 21033, 21031);
+
+-- Correct Items - Allyndia
+DELETE FROM `npc_vendor` WHERE `entry` = 4191 AND `item` IN (4536, 4537, 4538, 4539, 4602, 8953, 16171);
+UPDATE `npc_vendor` SET `maxcount` = 0, `incrtime` = 0 WHERE `entry` = 4191 AND `item` IN (21030, 21033, 21031);
+
+-- Striped Yellowtail - Heldan Galesong
+UPDATE `npc_vendor` SET `maxcount` = 0, `incrtime` = 0 WHERE `entry` = 4200 AND `item` = 21552;
+
+-- Rune Thread - Gorbold Steelhand
+DELETE FROM `npc_vendor` WHERE `entry` = 6301 AND `item` = 14341;
+
+-- Imbued Vial - Gorbold Steelhand
+DELETE FROM `npc_vendor` WHERE `entry` = 6301 AND `item` = 18256;
+
+-- Pattern: Murloc Scale Belt - Mavralyn
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 7200 WHERE `entry` = 4186 AND `item` = 5786;
+
+-- Pattern: Murloc Scale Breastplate - Mavralyn
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 7200 WHERE `entry` = 4186 AND `item` = 5787;
+
+-- Keen Throwing Knife - Dalmond
+DELETE FROM `npc_vendor` WHERE `entry` = 4182 AND `item` = 3107;
+
+-- Sharp Throwing Axe - Dalmond
+DELETE FROM `npc_vendor` WHERE `entry` = 4182 AND `item` = 3135;
+
+-- Pattern: Red Woolen Bag - Valdaron
+UPDATE `npc_vendor` SET `maxcount` = 3 WHERE `entry` = 4189 AND `item` = 5772;
+
+-- Pattern: Red Linen Bag - Valdaron
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 7200 WHERE `entry` = 4189 AND `item` = 5771;
+
+-- Pattern: Blue Linen Bag - Valdaron
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 7200 WHERE `entry` = 4189 AND `item` = 6270;
+
+-- Ashenvale
+-- Pattern: Herbalist's Gloves - Harloen Darkweave
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 10800 WHERE `entry` = 6731 AND `item` = 7361;
+
+-- Correct Items - Maiynn

```

Comment:
Comment says Maiynn, but creature id 4191 is Allyndia <Food & Drink Vendor>. Which one did you mean to update?